### PR TITLE
docs: note for current status of maintainers

### LINKED ISSUE #724: Archive Repository and Documentation
This repository is a public fork of https://github.com/FraunhoferISST/DataspaceConnector. Since it was last updated in 2022 and the upstream repository is archived in a read-only state, I would also vote to archive this repository.

Further, the documentation is still publicly available via the GitHub IO pages of the IDSA:
https://international-data-spaces-association.github.io/DataspaceConnector/
There isn't a hint that this documentation is outdated. Therefore, it leads to confusion for external people since it can be easily accessed and found via quick web searches. Different concepts and descriptions contrast with other documents, protocols, and specifications of the IDSA, like the dataspace protocol or the latest RAM.

### PR DESCRIPTION
**Pull request by ghost**
Notes have been added to the project files that this project is currently no longer actively maintained and is looking for new maintainers.

Closes #724 

_Note for reviewers: since the pipelines are failing, please merge this PR without waiting for requirements to be met and thus bypass branch protections._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,18 +119,22 @@ You are very welcome to contribute to this project when you find a bug, want to
 improvement, or have an idea for a useful feature. Please find a set of guidelines at the
 [CONTRIBUTING.md](CONTRIBUTING.md) and the [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md).
 
-The current development is driven by
+**This project is currently no longer maintained but looking for new maintainers.**
+
+Applications for maintainers shall be directed to IDSA.
```
Comment (ssteinbuss) [inline README.md]: IDSA is not unambiguous, please refer to International Data Space Association. 
Could you please add a contact link for reference, e.g. info@internationaldataspaces.org

File: README.md [hunk 2]
```diff
@@ -119,18 +119,22 @@ You are very welcome to contribute to this project when you find a bug, want to
...
+
+This project was previously maintained by Fraunhofer ISST and after that by sovity GmbH.
+Please do not approach former maintainers. If you have questions, plase open an issue / discussion for the community or approach the IDSA.
```
Comment (ssteinbuss) [inline README.md]: Please refer to International Data Spaces Association and provide a link to get in contact.

File: docs/index.md [hunk 3]
```diff
@@ -17,8 +19,13 @@ integrate it in your use cases, or contribute to its development.
 
 ---
 
-The Dataspace Connector is an IDS connector that is currently being maintained by [sovity](https://sovity.de/).
-The connector was originally developed at the Fraunhofer ISST.
+**This project is currently no longer maintained but looking for new maintainers.**
+
+Applications for maintainers shall be directed to IDSA.
```
Comment (ssteinbuss) [inline docs/index.md]: IDSA is not unambiguous, please refer to International Data Space Association.
Could you please add a contact link for reference, e.g. [info@internationaldataspaces.org](mailto:info@internationaldataspaces.org)

**Review by ssteinbuss — CHANGES_REQUESTED** [review]
Please provide a clear reference as described in the comments

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: done

File: docs/index.md (hunk 3 above)
Comment (ghost) [reply docs/index.md]: done

**Review by ssteinbuss — APPROVED** [review]
Thx for the quick changes

**Review by SebastianOpriel — APPROVED** [review]

**Pull request merged**